plugins/module_utils/aci.py

```diff
@@ -1592,7 +1597,14 @@ def exit_json(self, filter_existing=None, **kwargs):
         if "state" in self.params:
             self.original = self.existing
             if self.params.get("state") in ("absent", "present"):
-                self.get_existing()
+                if self.no_verification:
+                    if self.result["changed"]:
+                        self.result["current_verified"] = False
```

Comment:
What is the purpose of "current_verified"?

Reply:
Its only to inform the user that the current being shown has not been verified with APIC. I think it will become even more relevant when current becomes proposed. When I showed nothing on delete, current was empty because it gets cleared by using the new flag. However, when a user deletes an already deleted object, an empty current is actually what is returned from APIC hence, `current_verified=True`